otx/cli/tools/train.py

```diff
@@ -222,11 +238,8 @@ def main():
     assert resultset.performance is not None
     print(resultset.performance)
 
-    if args.save_logs_to:
-        tmp_path = task.project_path
-        logs_path = os.path.join(args.save_logs_to, tmp_path.split("/")[-1])
-        shutil.copytree(tmp_path, logs_path)
-        print(f"Save logs: {logs_path}")
+    if args.gpus:
```

Comment:
Can't we move this line to the #217?

Reply:
It can't. main process also needs to execute remained job. So, child processes should be killed after all jobs are done.